src/Libraries/DynamoUnits/DynamoUnits.Migrations.xml

```diff
@@ -3,29 +3,256 @@
   <priorNameHint>
     <oldName>DynamoUnits.SIUnit.Add@double</oldName>
     <newName>DSCore.Math.Add@double,double</newName>
+    <additionalAttributes>
+      <attribute>
+        <name>assembly</name>
+        <value>DSCoreNodes</value>
+      </attribute>
+    </additionalAttributes>
   </priorNameHint>
   <priorNameHint>
+    <oldName>DynamoUnits.SIUnit.Add@DynamoUnits.SIUnit</oldName>
+    <newName>DSCore.Math.Add@double,double</newName>
```

Comment:
Per our conversation on IM, is it possible to use the existing math operator nodes without having to introduce a new library?